src/Microsoft.AspNetCore.Mvc.Dnx/DnxAssemblyProvider.cs

```diff
@@ -1,88 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Reflection;
-using Microsoft.Extensions.PlatformAbstractions;
-
-namespace Microsoft.AspNetCore.Mvc.Infrastructure
-{
-    public class DnxAssemblyProvider : IAssemblyProvider
```

Comment:
Couldn't we just port this? The API doesn't seem significantly different and it avoids breaking people again.